src/Laserfiche.Repository.Api.Client.csproj

```diff
@@ -19,7 +19,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Laserfiche.Api.Client.Core" Version="1.2.0" />
+	<PackageReference Include="Laserfiche.Api.Client.Core" Version="1.2.0-beta-2790594917" />
```

Comment:
Fix tab white space to be consistent with other lines

Reply:
hmm indentation looks normal for me locally but i do see this issue remotely, so i guess i will make the change remotely thanks!